### ISSUE TITLE
error: package directory 'pulpo_forms' does not exist

### ISSUE THREAD

**Issue by pythonl1** [octobot-dev/pulpo-forms-django#1]
Hi,

I'm getting the above error while trying to install pulpo_forms using the python command ```python setup.py install ``` from the project directory.

Following is the output of command:

    `    Complete output from command python setup.py egg_info:
    running egg_info
    writing requirements to pulpo_forms.egg-info/requires.txt
    writing pulpo_forms.egg-info/PKG-INFO
    writing top-level names to pulpo_forms.egg-info/top_level.txt
    writing dependency_links to pulpo_forms.egg-info/dependency_links.txt
    warning: manifest_maker: standard file '-c' not found
    
    error: package directory 'pulpo_forms' does not exist
`

**Comment by elpaquete** [comment]
We are working on making a new up-to-date release in the upcoming days. It should fix this issues.
Have you tried using the pip install method instead?

**Comment by pythonl1** [comment]
Yes I got errors while installing with pip as well. Not the same error though.
